api/v1alpha1/nutanix_node_types.go

```diff
@@ -54,6 +54,10 @@ type NutanixMachineDetails struct {
 	// or using the prism_central API.
 	Subnets NutanixResourceIdentifiers `json:"subnets"`
 
+	// List of categories that need to be added to the machines. Categories must already exist in Prism Central
+	//+optional
```

Comment:
nit: I don't think it makes a difference, but I see most of the other optional fields have the space, plus my IDE complains about the no space
>Comment should start with a whitespace 
```suggestion
	// +optional
```